frontend/src/components/WofButton.vue

```diff
@@ -87,6 +87,21 @@ export default {
     }
   }
 
+  &--outline {
+    color: black;
+    background-color: transparent;
+    border: 1px solid @primary-dark-text-color;
+    &:hover {
+      background-color: fade(@primary-neutral-color, 30);
+      padding: calc(0.5rem + 1px) calc(1rem + 1px);
```

Comment:
What are you trying to do here?
This causes the text below the button to jump when you hover over the button.

Reply:
```vue
    <div>
      There is some text
      <wof-button type="outline">Text</wof-button>
      More text
    </div>
```

Reply:
I did it so that the text does not jump. Without this padding change it will jump

Reply:
Do I have to send a video? :P

Reply:
Ok i added now transparent border to every button. Now it will only change color, so it would not jump anymore

Reply:
It doesn't jump now!